### PR TITLE
Vendor in containers/buildah@v1.26.1 

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by rhatdan** [comment]
@flouthoc @Luap99 @vrothberg @giuseppe PTAL
@containers/podman-maintainers PTAL

**Review by vrothberg — COMMENTED** [review]
LGTM but we should fill out the change-log field

**Review by flouthoc — APPROVED** [review]
LGTM

**Comment by flouthoc** [comment]
@rhatdan `Build Each Commit` is failing with conflicts does it need a fresh `make vendor` PR instead of cherry-pick ?

**Comment by vrothberg** [comment]
I doesn't look like a vendor fart to me but something happens during a rebase:
```
Executing: hack/make-and-check-size /tmp/make-size-check.mOEtXud
Building: 7a0a3adc8 Add 4.1 branch to API documentation
make[1]: Entering directory '/var/tmp/go/src/github.com/containers/podman'
CGO_ENABLED=1 GOOS=linux GOARCH=amd64 go build \
	-mod=vendor  \
	-ldflags '-X github.com/containers/podman/v4/libpod/define.gitCommit=7a0a3adc83c2886d88a8c1a4795107417d8802ee -X github.com/containers/podman/v4/libpod/define.buildInfo=1651752017 -X github.com/containers/podman/v4/libpod/config._installPrefix=/usr/local -X github.com/containers/podman/v4/libpod/config._etcDir=/usr/local/etc -X github.com/containers/common/pkg/config.additionalHelperBinariesDir= ' \
	-tags "remote exclude_graphdriver_btrfs btrfs_noversion exclude_graphdriver_devicemapper containers_image_openpgp" \
	-o bin/podman-remote ./cmd/podman
if [ ! -x "/usr/bin/go-md2man" ]; then \
	make -C test/tools build/go-md2man ; \
fi
mkdir -p docs/build/man
make[1]: Leaving directory '/var/tmp/go/src/github.com/containers/podman'
bin/podman           size= 47609672  delta=     0
bin/podman-remote    size= 35758104  delta=     0
bin/rootlessport     size=  4889248  delta=     0
Rebasing (19/22)
Auto-merging go.mod
CONFLICT (content): Merge conflict in go.mod
Auto-merging go.sum
CONFLICT (content): Merge conflict in go.sum
Auto-merging vendor/modules.txt
CONFLICT (content): Merge conflict in vendor/modules.txt
error: could not apply 9ca88876f... Vendor in containers/buildah@v1.26.1
hint: Resolve all conflicts manually, mark them as resolved with
hint: "git add/rm <conflicted_files>", then run "git rebase --continue".
hint: You can instead skip this commit: run "git rebase --skip".
hint: To abort and get back to the state before "git rebase", run "git rebase --abort".
Could not apply 9ca88876f... Vendor in containers/buildah@v1.26.1
make: *** [Makefile:294: build-all-new-commits] Error 1
```

I do not yet see _why_ though.

**Comment by Luap99** [comment]
From the log at the top:
```
# Validate that all the commits build on top of origin/main^
git rebase origin/main^ -x "hack/make-and-check-size /tmp/make-size-check.mOEtXud"
```

Why would it base on origin/main? It should base it of origin/v4.1.

**Comment by vrothberg** [comment]
@rhatdan, apply the following patch:
```diff
diff --git a/Makefile b/Makefile
index caa991b140d5..bf60d26c8f91 100644
--- a/Makefile
+++ b/Makefile
@@ -32,7 +32,7 @@ HEAD ?= HEAD
 CHANGELOG_BASE ?= HEAD~
 CHANGELOG_TARGET ?= HEAD
 PROJECT := github.com/containers/podman
-GIT_BASE_BRANCH ?= origin/main
+GIT_BASE_BRANCH ?= origin/v4.1
 GIT_BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
 GIT_BRANCH_CLEAN ?= $(shell echo $(GIT_BRANCH) | sed -e "s/[^[:alnum:]]/-/g")
 LIBPOD_INSTANCE := libpod_dev
```

**Comment by Luap99** [comment]
> @rhatdan, apply the following patch:
> 
> ```diff
> diff --git a/Makefile b/Makefile
> index caa991b140d5..bf60d26c8f91 100644
> --- a/Makefile
> +++ b/Makefile
> @@ -32,7 +32,7 @@ HEAD ?= HEAD
>  CHANGELOG_BASE ?= HEAD~
>  CHANGELOG_TARGET ?= HEAD
>  PROJECT := github.com/containers/podman
> -GIT_BASE_BRANCH ?= origin/main
> +GIT_BASE_BRANCH ?= origin/v4.1
>  GIT_BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
>  GIT_BRANCH_CLEAN ?= $(shell echo $(GIT_BRANCH) | sed -e "s/[^[:alnum:]]/-/g")
>  LIBPOD_INSTANCE := libpod_dev
> ```

Note sure if this is the right solution, looks like DEST_BRANCH is not set correctly
https://github.com/containers/podman/blob/8081d921b2bf39a418155a6822e55f3392ad63f6/.cirrus.yml#L9 
https://github.com/containers/podman/blob/09b8831fa21c47bf1dac05381b61cec0092f94c5/contrib/cirrus/runner.sh#L261

File: go.mod
Comment (vrothberg) [inline go.mod:18]: It bothers me to vendor a non-release. I'll check where this comes from.

Comment (vrothberg) [reply go.mod:18]: @rhatdan can you vendor v1.4.0 (https://github.com/containers/ocicrypt/releases/tag/v1.1.4)?

**Comment by mheon** [comment]
Fair enough, seems reasonable to try.

@rhatdan Can you repush with the requested changes, or should I take this over?

**Comment by mheon** [comment]
I'm going to
/lgtm
/approve
/hold

So I can quickly get 4.1.0 final out

**Pull request merged**
